### PR TITLE
CONTROL_TRANSFER_EX implemented

### PR DESCRIPTION
**Pull request by koudis**
URB_FUNCTION_CONTROL_TRANSFER_EX implemented.

I tested it on my Unyfing and USB devices.

### CODE REVIEW

File: driver/vhci/vhci_read.c [hunk 1]
```diff
@@ -519,6 +519,65 @@ store_urb_iso(PIRP irp, PURB urb, struct urb_req *urbr)
 	return STATUS_SUCCESS;
 }
 
+static NTSTATUS
+store_urb_control_transfer_ex_partial(pusbip_vpdo_dev_t vpdo, PIRP irp, PURB urb)
+{
+	struct _URB_CONTROL_TRANSFER_EX* urb_control_ex = &urb->UrbControlTransferEx;
+	PVOID	dst;
+	char* buf;
```
Comment (cezanne) [inline driver/vhci/vhci_read.c]: I made a diff for tab+asterisk. 😃 Please refer [tab_asterisk_diff.txt](https://github.com/cezanne/usbip-win/files/4333298/tab_asterisk_diff.txt).

**Comment by koudis** [comment]
@cezanne ok, I will repair CS violations :)

**Comment by koudis** [comment]
Repaired.

**Comment by cezanne** [comment]
@koudis : Your work has been merged into the master. Thanks for your efforts.

**Pull request merged**